package.json

```diff
@@ -112,12 +112,12 @@
     "eslint-plugin-flowtype": "^2.35.1",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-prettier": "^2.2.0",
-    "eslint-plugin-react": "^7.3.0",
+    "eslint-plugin-react": "^7.4.0",
```

Comment:
Upgrading babel and eslint stuff wasn't strictly necessary, tell me if you'd like this to move to another PR.

Reply:
I don't mind.